mne/viz/epochs.py

```diff
@@ -93,319 +94,376 @@ def plot_epochs_image(epochs, picks=None, sigma=0., vmin=None,
+    from matplotlib.pyplot import figure, Axes, subplot2grid
+
+    n_axes = 1 + int(evoked) + int(colorbar)
+    ax_names = ('image', 'evoked', 'colorbar')
+    ax_names = np.array(ax_names)[np.where([True, evoked, colorbar])]
+    prefix = 'Since evoked={} and colorbar={}, '.format(evoked, colorbar)
+
+    # got both fig and axes
+    if fig is not None and axes is not None:
+        raise ValueError('At least one of "fig" or "axes" must be None; got '
+                         'fig={}, axes={}.'.format(fig, axes))
+
+    # got fig=None and axes=None: make fig(s) and axes
+    if fig is None and axes is None:
+        axes = dict()
+        colspan = 9 if colorbar else 10
+        rowspan = 2 if evoked else 3
+        shape = (3, 10)
+        for this_group in group_by:
+            this_fig = figure(this_group)
+            subplot2grid(shape, (0, 0), colspan=colspan, rowspan=rowspan,
+                         fig=this_fig)
+            if evoked:
+                subplot2grid(shape, (2, 0), colspan=colspan, rowspan=1,
+                             fig=this_fig)
+            if colorbar:
+                subplot2grid(shape, (0, 9), colspan=1, rowspan=rowspan,
+                             fig=this_fig)
+            axes[this_group] = this_fig.axes
+
+    # got a Figure instance
+    if fig is not None:
+        # If we're re-plotting into a fig made by a previous call to
+        # `plot_image`, be forgiving of presence/absence of sensor inset axis.
+        if len(fig.axes) not in (n_axes, n_axes + 1):
+            raise ValueError('{}"fig" must contain {} axes, got {}.'
+                             ''.format(prefix, n_axes, len(fig.axes)))
+        if len(list(group_by)) != 1:
+            raise ValueError('When "fig" is not None, "group_by" can only '
+                             'have one group (got {}: {}).'
+                             .format(len(group_by), ', '.join(group_by)))
+        key = list(group_by)[0]
+        if clear:  # necessary if re-plotting into previous figure
+            _ = [ax.clear() for ax in fig.axes]
+            if len(fig.axes) > n_axes:  # get rid of sensor inset
+                fig.axes[-1].remove()
+            fig.canvas.set_window_title(key)
+        axes = {key: fig.axes}
+
+    # got an Axes instance, be forgiving (if evoked and colorbar are False)
+    if isinstance(axes, Axes):
+        axes = [axes]
+
+    # got an ndarray; be forgiving
+    if isinstance(axes, np.ndarray):
+        axes = axes.ravel().tolist()
+
+    # got a list of axes, make it a dict
+    if isinstance(axes, list):
+        if len(axes) != n_axes:
+            raise ValueError('{}"axes" must be length {}, got {}.'
+                             ''.format(prefix, n_axes, len(axes)))
+        # for list of axes to work, must be only one group
+        if len(list(group_by)) != 1:
+            raise ValueError('When axes is a list, can only plot one group '
```

Comment:
Missing a comma?

Reply:
I don't think so, would you put a comma right before parentheses, (I wouldn't)?